src/test/test-seccomp.c

```diff
@@ -1006,6 +1007,23 @@ static int real_open(const char *path, int flags, mode_t mode) {
 #endif
 }
 
+static int try_fchmodat2(int dirfd, const char *path, int flags, mode_t mode) {
```

Comment:
maybe move this into missing_syscall.h and then start using this in fchmod_opath(), at two places in copy.c, in rm-rf.c, sysupdate-transfer.c

Reply:
That's a great suggestion; it can be done in a future patchset, I guess let's get this in first.

Reply:
> That's a great suggestion; it can be done in a future patchset, I guess let's get this in first.

I've just filed #29664 to track this.